docs/bootstrap.sh

```diff
@@ -37,7 +37,7 @@ function release_docs {
   yarn install
   yarn build
 
-  if ! deploy_output=$(yarn netlify deploy --site aztec-docs-dev --prod 2>&1); then
+  if ! deploy_output=$(yarn netlify deploy --site aztec-docs-dev --prod); then
```

Comment:
do we not want to capture error output? or its already going to be logged?

Reply:
Hmmm yeah this logic kinda depends on the silence when it works. Should maybe refactor to make it just print regardless of whether it's successful or errors